SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/visitors/ThreeOperandVisitor.java

```diff
@@ -47,6 +47,36 @@
  * };
  */
 public class ThreeOperandVisitor extends InnerVisitor implements Function<IDBSPInnerNode, IDBSPInnerNode>, IModule {
+    static class PendingStatements {
```

Comment:
This sounds like it should be called a block?

Reply:
It will eventually become the contents of a block.